Formula/srt.rb

```diff
@@ -0,0 +1,23 @@
+class Srt < Formula
+  desc "Secure Reliable Transport"
+  homepage "https://www.srtalliance.org/"
+  url "https://github.com/Haivision/srt/archive/v1.2.2.tar.gz"
+  sha256 "7a167cfe9f3a497c8d9483f25873fe63273c550aaee1d2bbb9d7bf39f169efcc"
+  head "https://github.com/Haivision/srt.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
+
+  def install
+    openssl = Formula["openssl"]
+    system "cmake", ".", "-DWITH_OPENSSL_INCLUDEDIR=#{openssl.opt_include}",
+                         "-DWITH_OPENSSL_LIBDIR=#{openssl.opt_lib}",
+                         *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/stransmit", "file:///dev/null", "file://con/"
```

Comment:
The test should assert something here beyond just a zero exit code. See `dep`, `jose`, and `faas-cli` for some examples.

Reply:
I don't think the current SRT has any test for doing that kind of test.
`stransmit` requires UDP or SRT streaming packets as input, not file. 
Unfortunately, even though a test program sends UDP packets to `stransmit`, there's no way to compare with what the output is.

Another possible way would be to write some testing codes, but that's out of the scope of homebrew formula. It should be discussed in SRT.